src/main/java/nl/knaw/dans/filemigration/core/EasyFileLoader.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.filemigration.core;
+
+import nl.knaw.dans.filemigration.api.EasyFile;
+import nl.knaw.dans.filemigration.api.ExpectedFile;
+import nl.knaw.dans.filemigration.db.EasyFileDAO;
+import nl.knaw.dans.filemigration.db.ExpectedFileDAO;
+import org.hibernate.exception.ConstraintViolationException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.persistence.PersistenceException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class EasyFileLoader {
+  private static final Logger log = LoggerFactory.getLogger(EasyFileLoader.class);
+
+  private final EasyFileDAO easyFileDAO;
+  private final ExpectedFileDAO expectedDAO;
+
+  public EasyFileLoader(EasyFileDAO easyFileDAO, ExpectedFileDAO expectedDAO) {
+    this.expectedDAO = expectedDAO;
+    this.easyFileDAO = easyFileDAO;
+  }
+
+  public void loadFromCsv(FedoraToBagCsv csv) {
+    if (!csv.getComment().contains("OK"))
+      log.warn("skipped {}", csv);
+    else {
+      if (!csv.getComment().contains("no payload"))
+        fedoraFiles(csv);
+      Arrays.stream(migrationFiles).iterator()
+          .forEachRemaining(f -> saveExpected(addedMigrationFile(csv, f)));
+    }
+  }
+
+  /** note: bag-to-deposit also adds emd.xml to bags from the vault, that does not apply to migration */
+  private static final String[] migrationFiles = { "provenance.xml", "dataset.xml", "files.xml" };
```

Comment:
```suggestion
  private static final String[] migrationFiles = { "provenance.xml", "dataset.xml", "files.xml", "emd.xml" };
```

Reply:
Whoops:
https://github.com/DANS-KNAW/easy-convert-bag-to-deposit/blob/ec42d59bd7e85be0ebe48f0a53d4c65e8369436b/src/main/scala/nl.knaw.dans.easy.bag2deposit/EasyConvertBagToDepositApp.scala#L186

Reply:
anyway, we are handling files from fedora in this class, not from the vault.